src/main/webapp/css/style.css

```diff
@@ -22,3 +22,32 @@ h1 {
 .green {
   color: #0F9D58;
 }
+
+.header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between; 
+  margin-left: 48%;
+}
+
+.auth-button {
+  right: 8px;
+  font: 400 13px Arial;
+  color: white;
+  background: #4d90fe;
+  border: 1px #3079ED solid;
+  padding: 5px 10px;
+  border-radius: 2px;
+  text-align: center;
```

Comment:
This does nothing since a is an inline element, and text-align works on block elements (and its descendants).